Exercise_C.js

```diff
@@ -13,6 +13,17 @@ If the number is divisible by 3 and 5 it should log Hong Kong.
 See README.md if you need help running your code.
 */
 
-for (let i = 1; i <= 30; i++) {
-  // CODE HERE
-}
+let output = [];
+
+for (let i = 1;i <= 30; i++) {
+    if (((i % 3) == 0) && ((i % 5) == 0))
+     output.push("Hong Kong")
+    else if ((i % 3) == 0)
+      output.push("Hong");
+    else if ((i % 5) == 0)
+      output.push("Kong");
+     else
+        output.push(i);
+  }
+
+console.log(output);
```

Comment:
Amazing Work! This works correctly, as you have first run a loop until the number 30 and made use of Conditionals to check the remainder and also used the % syntax.